java/vector/src/test/java/org/apache/arrow/vector/TestDictionaryVector.java

```diff
@@ -88,8 +88,9 @@ public void testEncodeStrings() {
 
       Dictionary dictionary =
```

Comment:
we should have additional tests for the new APIs instead of modifying existing one.

Reply:
Thanks, my bad, forgot to revert this. I will fix this later today :).

Reply:
fixed now.